builds/checkin/edgelet.yaml

```diff
@@ -54,6 +59,11 @@ phases:
         inputs:
           filePath: edgelet/build/linux/cross-test.sh
           arguments: --toolchain armv7-unknown-linux-gnueabihf
+      - task: Bash@3
+        displayName: Clean
+        inputs:
+          targetType: inline
+          script: '$HOME/.cargo/bin/cargo clean'
```

Comment:
Is this just for testing? Should we have a cleanup script, or is this solid enough?

Reply:
Yes, it should be a script, this is way more awkward than I'd like.